src/helpers/helpers.js

```diff
@@ -0,0 +1,19 @@
+export const fieldRemover = loggedInUser => {
```

Comment:
can you find a better name for this file?
helpers.js is a bit confusing ...

Reply:
@espoirMur Can you suggest one. Thanks